### PR TITLE
Fix issue with System.Net.Http reference

### PR DESCRIPTION
**Pull request by jdom**
In my Windows Server box (no development SDK installed), the System.Net.Http reference wasn't automatically picked up from the GAC, so it needed to be copied to the bin output folder

### CODE REVIEW

**Comment by VorlonCD** [comment]
Thanks!    Interesting, did you have a bad install of .net framework 4.7.2 perhaps?

**Comment by jdom** [comment]
TL;DR: you might undo the Private=True on System.Net.Http if you want
Hmmm, I'm not sure. I'm trying to repro again (by deleting the System.Net.Http.dll) to nail it down, but can't any more. I did notice something odd at some point yesterday when copying everything again, that the AiTool.exe.config was very different in size when I was having this issue, so perhaps for some reason the first time it generated the wrong config, and it was just an assembly binding issue. I didn't have Fusion log viewer in my server to troubleshoot the loading issue, so I don't know the reason why that assembly wasn't loading from the GAC.
Feel free to undo that change. Still, writing the error to the log was useful for troubleshooting, as at first it was just silently failing with nothing useful to understand the issue. So please don't undo that portion.

**Comment by VorlonCD** [comment]
I didnt mind merging those changes anyway.   Make sure you see them with my push from yesterday.   I think I got them all.

Sometimes when a .net framework windows update installs, it breaks framework until reboot.  Maybe something like that happened?

**Comment by jdom** [comment]
Yeah, I saw the changes, thanks for merging. I'm building locally also anyway. But now I removed the System binary and it's still breaking. My theory is that I had a broken app.config, and hence no binding redirect was working (I didn't update nor restart anything in the Windows Server). It's all fine now.

I like the update-momentum, I might try to find some time to contribute. One thing I'd like is to enable face detection too, and do not alert if it sees me or my family, so I don't have to constantly switch to a profile without notifications, to avoid getting spammed in Telegram when we are outside. TBH I haven't played with the cooldown features much... I'm a little worried about missing some alerts if I turn it on (although with the merged tagging of where the person is that you added, I'll give it a try, since I can now see exactly what deepstack tagged and loose more closely at that area).

**Comment by VorlonCD** [comment]
If you want to start working on the guts for face detection (an easy way to train it, etc) that would be great!  I would start off using some of the HttpClient code in DetectObjects.

@GentlePumpkin started working on face detection in another branch of his I think.  Havent looked into how far he got.
Note that the HISTORY class has a few things that I cooked in for the future - IsFace and IsKnownFace.  Guess prob need to add a name somewhere too.
